reactions/src/HALO.cpp

```diff
@@ -913,15 +949,70 @@ void HALO::react_init_nu2(double vars[], Spline ploopr, Spline ploopp, bool mgca
 }
 
 
-// Calculate the reaction with massive neutrinos
+/* The 1-loop SPT reaction */
+
+double HALO::reaction_spt(double k0, double vars[], bool mgcamb) const{
```

Comment:
What is this used for? I can't find an example.

Reply:
We use SPT at 0.06h/Mpc so it should agree exactly with the halo model reaction there. I used this function to test that for the MG+neutrino cases. It's not needed really, but might be useful to keep.

Reply:
Ok, thanks.